assets/src/blocks/Gallery/GalleryBlock.js

```diff
@@ -63,23 +63,23 @@ export class GalleryBlock {
         {
           name: 'slider',
           label: getStyleLabel(
-            'Slider',
-            'The slider is a carousel of images. For more than 5 images, consider using a grid.'
+            __('Slider', 'planet4-blocks-backend'),
```

Comment:
I noticed this issue for the Submenu block, so I made these changes here as well already.